training/src/components/RadioGroup/index.js

```diff
@@ -1,3 +1,3 @@
 import RadioGroup from './RadioGroup';
 
-export default RadioGroup;
+export { RadioGroup };
```

Comment:
instead of this you can simple use

`export { default as RadioGroup } from './RadioGroup';`